src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -565,6 +566,34 @@ const DECISION_REVIEW_CONTENT = (
   </div>
 );
 
+/**
+ * Translates an array of two ints into a string that conveys a duration estimate
+ * @param {number[]} timeliness two integers that represent the low and high time durations
+ * (in months) of a given thing
+ * @returns {string} formatted to convey the estimated duration range, in months
+ */
+const makeDurationHeader = (timeliness) => {
+  if (!timeliness || !Array.isArray(timeliness) || timeliness.length !== 2) {
```

Comment:
Not sure we need this, open to thoughts.

Reply:
Eh, I this it's good.